.github/workflows/ruby.yml

```diff
@@ -11,9 +11,7 @@ jobs:
     - name: Check out code
       uses: actions/checkout@v2
     - name: Set up Ruby
-      uses: actions/setup-ruby@v1.1.3
-      with:
-        ruby-version: '2.7.2'
+      uses: ruby/setup-ruby@v1
     - name: Restore cache
       uses: actions/cache@v2.1.4
       with:
```

Comment:
I think we should be able to get rid of actions/cache and the "Install dependencies" step with the new action: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby#caching-bundle-install-automatically